Dockerfile

```diff
@@ -0,0 +1,21 @@
+ARG PYTHON_VERSION=3.8
+
+FROM python:${PYTHON_VERSION} AS base
+
+WORKDIR /app
+
+COPY . ./
+
+FROM python:${PYTHON_VERSION}-slim
+
+WORKDIR /app
+
+ENV PYTHONUNBUFFERED=1
+
+COPY --from=base /app .
+
+RUN pip install -r requirements.txt && python manage.py migrate
+
+EXPOSE 8080
+
+ENTRYPOINT ["python","manage.py","runserver","0.0.0.0:8080"]
```

Comment:
![image](https://github.com/mate-academy/devops_todolist/assets/87720583/d3e98657-f427-4d2d-a9b6-29d8b5236fcb)
Add Eof(End-of-file), need to add empty line at the end